### DISCUSSION TITLE
Doris如何获取表的索引？

### DISCUSSION THREAD

**Discussion by MengXiangDing** [apache/doris#26450] [A - General / Q&A] [answered]
例如像 MySQL 这样：show index from table from schema; show keys from table from schema;
但是 Doris 执行这种语句，得出的结果是空的

**Comment by zhbdesign** [comment]
SHOW BUILD INDEX;

**Reply by MengXiangDing** [reply]
Thank you for your answer, but I still can't seem to get it
<img width="589" alt="image" src="https://github.com/apache/doris/assets/73006046/d29c673e-106b-4955-8643-92c6e11a8067">

**Comment by MengXiangDing** [comment, marked as answer]
研究了下，可以用这个 sql 获取 unique index
select * from information_schema.columns where table_name = 'table' and table_schema = 'schema' and column_key = 'UNI'
